pages/about.vue

```diff
@@ -61,16 +61,17 @@
             >
                 <div
                     v-for="(member, index) in data.members"
-                    :key="index"
+                    :key="member.avatarImg"
                     data-testid="member"
                     class="members-list grid"
                 >
                     <MemberComponent
                         :avatar-img="member.avatarImg"
+                        :github-url="member.githubUrl"
+                        :linked-in-url="member.linkedInUrl"
                         :name="member.name"
+                        :test-number="index"
```

Comment:
I wouldn't know from reading this property what it does. maybe `data-testid` would be better

Reply:
I have a question though. Would that not be confusing since `data-testid` is a cypress thing. And this is just to pass the index down to be used later in the data-testid? It by itself is not the test Id but being used as an identifier in the test id. Would having an element attribute and a prop named the same thing not be more confusing? Is there another name you would suggest @ermish ?

Reply:
`data-testid` is just the name I use for tests since it's generic and can be used by any testing library. 
[Cypress](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements) usually prefers `data-cy`. It's fine to have a prop named that makes it clear what it's being used for. I would just call it `dataTestId` which would be easy to understand but is technically different.